src/routes.tsx

```diff
@@ -104,7 +103,6 @@ export const routes: RouteObject[] = [
             {
               path: '/search', // Global search
               element: withDefaultTheme(<SearchResultsPage />),
-              loader: searchParamLoader,
```

Comment:
Why are we removing the `searchParamLoader` here? I would rather see that app fails in a loader if invalid URL, instead of in component.

Reply:
I don't think searchParamLoader was validating the URL. We just needed to parse the query in URL, but async code forced us to use a loader (or useEffects)

Reply:
Should have linked to master, either way, this code was previously in `searchParamLoader`

Reply:
Checked more thoroughly

Before:
- /:_network/:_layer/search:
  - /:_network/:_layer/: assertEnabledScope validated network and scope
  - /:_network/:_layer/search: searchParamLoader validated network again
- /search: searchParamLoader skipped the validation because !!network is false

After:
- /:_network/:_layer/search:
  - /:_network/:_layer/: assertEnabledScope validates network and scope
- /search: nothing to validate

Reply:
(found https://github.com/oasisprotocol/explorer/pull/1366 while checking)